logfile

```diff
@@ -0,0 +1,2 @@
+2019-02-11 10:48:40.115 EST [76779] FATAL:  lock file "postmaster.pid" already exists
+2019-02-11 10:48:40.115 EST [76779] HINT:  Is another postmaster (PID 501) running in data directory "/usr/local/var/postgres"?
```

Comment:
Looks like a stray logfile got added by mistake.

Reply:
probably that was mine